.gitattributes

```diff
@@ -1,4 +1,4 @@
 * text=auto eol=lf
-*.ai binary
+*.snap linguist-generated=true
```

Comment:
Side note: `=true` can be omitted ~~like in HTML~~

Reply:
```suggestion
*.snap linguist-generated
```